src/Console/Command/HelpCommand.php

```diff
@@ -449,6 +450,10 @@ function (FixerInterface $a, FixerInterface $b) {
                 $description = '[n/a]';
             }
 
+            if ($fixer instanceof DeprecatedFixerInterface) {
```

Comment:
this condition is needed also for `DescribeCommand`

Reply:
@keradus done